### PR TITLE
Optimise using RUN

### PR DESCRIPTION
**Pull request by vasiliy-zaznobin**
Optimize using RUN COPY
Remove debug steps

### CODE REVIEW

**Comment by vasiliy-zaznobin** [comment]
Full build passed in 5m 43s 
https://buildkite.com/cyberway/neon-proxy/builds/16

Increment build passed in 4m 3s
https://buildkite.com/cyberway/neon-proxy/builds/17

File: Dockerfile [hunk 1]
```diff
@@ -28,19 +26,25 @@ COPY --from=cli /opt/solana/bin/solana \
...
+COPY ./requirements.txt /opt
+WORKDIR /opt
+
+RUN python3 -m venv venv \
+    && \
+    pip3 install --upgrade pip \
+    && \
+    /bin/bash -c "source venv/bin/activate" \
+    && \
+    pip install -r requirements.txt
```
Comment (s-medvedev) [inline Dockerfile]: I would suggest not to wrap `&&` sign on another line and do it like:
```
RUN python3 -m venv venv && \
    pip3 install --upgrade pip && \
    /bin/bash -c "source venv/bin/activate" && \
    pip install -r requirements.txt
```

File: Dockerfile [hunk 2]
```diff
@@ -4,20 +4,18 @@ FROM cybercoredev/evm_loader:latest AS spl
...
+    software-properties-common \
+    openssl \
+    ca-certificates \
+    curl \
+    python3-pip \
+    python3-venv \
+    && \
+    rm -rf /var/lib/apt/lists/*
 
 COPY --from=cli /opt/solana/bin/solana \
```
Comment (s-medvedev) [inline Dockerfile:21]: You can do all this COPY after install requirements.txt.

File: Dockerfile [hunk 3]
```diff
@@ -4,20 +4,18 @@ FROM cybercoredev/evm_loader:latest AS spl
...
+    DEBIAN_FRONTEND=noninteractive \
+    apt -y install \
+    software-properties-common \
+    openssl \
+    ca-certificates \
+    curl \
+    python3-pip \
+    python3-venv \
+    && \
+    rm -rf /var/lib/apt/lists/*
```
Comment (s-medvedev) [inline Dockerfile]: Maybe reformat like this?
```
RUN apt update && \
    DEBIAN_FRONTEND=noninteractive apt -y install \
               software-properties-common openssl ca-certificates \
               curl python3-pip python3-venv && \
    rm -rf /var/lib/apt/lists/*
```

Comment (vasiliy-zaznobin) [reply Dockerfile]: OK. Done

File: Dockerfile (hunk 2 above)
Comment (vasiliy-zaznobin) [reply Dockerfile:21]: OK. Done

File: Dockerfile (hunk 1 above)
Comment (vasiliy-zaznobin) [reply Dockerfile]: OK. Done

File: Dockerfile (hunk 3 above)
Comment (vasiliy-zaznobin) [reply Dockerfile]: Full build passed in 5m 46s
https://buildkite.com/cyberway/neon-proxy/builds/19

Increment build passed in 4m 4s
https://buildkite.com/cyberway/neon-proxy/builds/20

**Review by s-medvedev — APPROVED** [review]

**Pull request merged**